### PR TITLE
enable separate exports (to support vite usecase)

### PR DESCRIPTION
**Pull request by jnsdls**

### CODE REVIEW

**Comment by jnsdls** [comment]
this will enable a workaround fix for #458 by enabling users to import the correct version of the SDK directly

For browser usage (where the bundler does not pick it up correctly) (vite):
```js
import { ThirdwebSDK } from "@thirdweb-dev/sdk/dist/browser";
```

File: package.json [hunk 1]
```diff
@@ -13,12 +13,32 @@
   "browser": "dist/browser/index.js",
   "unpkg": "dist/browser/index.global.js",
   "exports": {
-    "node": {
+    ".": {
```
Comment (jnsdls) [inline package.json:16]: "root" level exports

File: package.json [hunk 2]
```diff
@@ -13,12 +13,32 @@
...
+    "./dist/node": {
+      "types": "./dist/node/index.d.ts",
       "import": "./dist/node/index.mjs",
       "module": "./dist/node/index.mjs",
-      "require": "./dist/node/index.js"
+      "require": "./dist/node/index.js",
+      "default": "./dist/node/index.js"
     },
-    "default": "./dist/browser/index.mjs"
+    "./package.json": "./package.json"
```
Comment (jnsdls) [inline package.json:41]: convention when doing file-level exports to add this

File: package.json [hunk 3]
```diff
@@ -13,12 +13,32 @@
...
+        "types": "./dist/node/index.d.ts",
+        "import": "./dist/node/index.mjs",
+        "module": "./dist/node/index.mjs",
+        "require": "./dist/node/index.js",
+        "default": "./dist/node/index.js"
+      },
+      "types": "./dist/browser/index.d.ts",
+      "default": "./dist/browser/index.mjs"
+    },
+    "./dist/browser": {
```
Comment (jnsdls) [inline package.json:27]: lets u import `dist/browser` directly

**Review by joaquim-verges — COMMENTED** [review]
works on node 🤷‍♂️

**Pull request merged**